src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -144,11 +144,11 @@ def arch_repo_table_cell(repo, arch, packname)
     'building'    => 'Build jobs exists',
     'finished'    => 'Build jobs have been processed, new repository is not yet created',
     'blocked'     => 'No build possible atm, waiting for jobs in other repositories',
-    'broken'      => 'The repository setup is broken, build not possible',
+    'broken'      => 'The repository setup is broken, build or publish not possible',
     'scheduling'  => 'The repository state is being calculated right now'
   }
 
-  def repo_status_icon(status)
+  def repo_status_icon(status, details = nil)
```

Comment:
no one yet. mentioned in the commit message, it is preparation for showing the error in the webui to fix the issue. But I have not touched the UI yet.